src/main/java/ru/itis/controllers/ProfileController.java

```diff
@@ -50,8 +50,8 @@ public Object createNewWL(@RequestParam String title, @RequestHeader("Authorizat
     @ApiOperation("Delete a wish list")
     public Object deleteWishList(@RequestParam String title, @RequestHeader("Authorization") String token) {
```

Comment:
Return type should be changed to `ResponseEntity<WishList>`

Reply:
fixed to ResponseEntity<?>